src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = (number) => {};
-
+const numberToReversedDigits = (number) => {
+  const reversedArray = Array.from(String(number), n => Number(n));
+  reversedArray.reverse();
+  return reversedArray;
+};
```

Comment:
You can reduce this down a bit by not declaring the variables at all, and just returning the chain of existing logic together. Dos the same job in a bit less space while still being readable:
```
const numberToReversedDigits = (number) =>
  Array.from(String(number), (n) => Number(n)).reverse();
```